lib/rules/id-length.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Rule that warns when identifier names are shorter or longer
+ *               than the values provided in configuration.
+ * @author Burak Yigit Kaya aka BYK
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function (context) {
+    var options = context.options[0] || {};
+    var minLength = typeof options.min === "number" ? options.min : 2;
```

Comment:
Don't need to check for number, schema is going to blow up if non-number is going to be passed in.